Only use django jQuery

leibowitz opened the pull request:
The `bindfields.js` seems to fail to receive the `formset:added` event, probably because those events are triggered using the `django.jQuery`.
https://docs.djangoproject.com/en/1.11/ref/contrib/admin/javascript/#inline-form-events

leibowitz commented:
I just realised this is working as well simply by adding `USE_DJANGO_JQUERY = True` to the settings.

manelclos commented:
@leibowitz I can imagine the setting is there because at times you would need a different jQuery version.

Can we close this PR?

leibowitz commented:
Even if that breaks anything that relies on the `formset:added` and `formset:removed` events? I'm not sure it's a good idea to allow to customise the jQuery version, if that will lead to other issues. Or am I missing something?

manelclos commented:
I agree. This can be a source of problems. If we apply your PR though, it also needs to contain removal of the two options that allow jQuery version customisation: `USE_DJANGO_JQUERY` and `JQUERY_URL `. And the documentation should state that these options have been removed since version x.y.z.

Could you please add that to the PR?

leibowitz commented:
Sure, going to create a new PR as this is quite different
https://github.com/jazzband/django-smart-selects/pull/301

Regarding this,
> And the documentation should state that these options have been removed since version x.y.z.

It should appear in the changelog, but there's none. Time to create one?
